modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesKernelBoot.java

```diff
@@ -450,4 +451,8 @@ private static void createSuperTenantDLC() throws AndesException {
             throw new AndesException("Error getting super tenant username", e);
         }
     }
+
+    public static boolean isKernelShuttingDown() {
```

Comment:
Since this is a de-facto getter method, the intention is to refer to the member variable comment for information.